drivers/bus/pcix/utils.c

```diff
@@ -1330,6 +1328,9 @@ PciQueryBusInformation(IN PPCI_PDO_EXTENSION PdoExtension,
     BusInfo->BusTypeGuid = GUID_BUS_TYPE_PCI;
     BusInfo->LegacyBusType = PCIBus;
     BusInfo->BusNumber = PdoExtension->ParentFdoExtension->BaseBus;
+
+    *Buffer = BusInfo;
```

Comment:
This change implicates the need to adjust the SAL for that variable (line 1317).